modules/learn/pages/security/certificates.adoc

```diff
@@ -192,6 +193,8 @@ Each specified _path_ can be one of the following path-types:
 
 * `subject.cn`.
 The _Subject Common Name_ specified in the certificate will be extracted.
+Note that use of Subject Common Name is now deprecated.
```

Comment:
It sounds as it is deprecated in Couchbase Server. If I were a customer I would decide that I need to regenerate my certs, but that's not true. Customers can actually continue to use CN, at least right now we don't have any plans to remove support for username in CN for client certs.
In my opinion we can mention that it's deprecated in rfc6125 but we still support it.

Reply:
I've added a link to the RFC, and a statement that we still support it.

Reply:
I've also moved this to a place a little earlier in the section, where client's use of cn is first described.